### PR TITLE
refactor: rename to @imgix/ng and imgix-ng internally

### PR DESCRIPTION
**Pull request by frederickfogerty**
Tested that the app builds and the example app still works.

### CODE REVIEW

**Review by sherwinski — APPROVED** [review]
No code changes needed, but we should hold off on merging this until we can discuss it offline.

**Comment by frederickfogerty** [comment]
Hey @sherwinski I'm going to merge this for now after our discussion from yesterday. We can always update the naming later if needed

**Pull request merged**
